src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -958,25 +958,31 @@ public static Vec3d rayTraceEyeHitVec(EntityLivingBase entity, double length)
         return git == null ? null : git.hitVec;
     }
 
-    public static boolean onInteractEntityAt(EntityPlayer player, Entity entity, RayTraceResult ray, ItemStack stack, EnumHand hand)
```

Comment:
This breaks binary compatibility with any mod that calls these functions.
I've already stated this NEEDs to be fixed.